e2e/test/scenarios/question-reproductions/reproductions-3.cy.spec.js

```diff
@@ -1737,6 +1739,238 @@ describe("issue 39771", () => {
   });
 });
 
+describe("issue 45063", () => {
```

Comment:
I don't see a repro for #7231

Reply:
Let's add a `(metabase#7231)` tag somewhere (`cy.log`?) for it.
Also for #37955